src/widget/wsearchlineedit.cpp

```diff
@@ -217,31 +211,35 @@ void WSearchLineEdit::setup(const QDomNode& node, const SkinContext& context) {
     setPalette(pal);
 
     m_clearButton->setToolTip(tr("Clear input") + "\n" +
-            tr("Clear the search bar input field") + "\n\n" +
-
-            tr("Shortcut") + ": \n" +
-            tr("Ctrl+Backspace"));
+            tr("Clear the search bar input field"));
+}
 
+void WSearchLineEdit::setupToolTip(const QString& searchInCurrentViewShortcut,
```

Comment:
this is not called?

Reply:
Good catch! The changes to LegacySkinParser got lost during cherry-picking the changes for this PR... should be fixed now.